l10n_br_sale/hooks.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2020  Renato Lima - Akretion <renato.lima@akretion.com.br>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+import logging
+
+from odoo import _, api, tools, SUPERUSER_ID
+
+_logger = logging.getLogger(__name__)
+
+
+def post_init_hook(cr, registry):
+
+    if not tools.config['without_demo']:
+        _logger.info(_(
+            "Loading l10n_br_sale demo files. It may take at"
+            " least a minutes...")
+        )
```

Comment:
that log is a left over the l10n_br_fiscal hook and makes no sense here I think... (this hook is fast)